### PR TITLE
EPMRPP-46985 Add item attributes support

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: src/realTimeReporter/reportingApi/attachData.ts [hunk 1]
```diff
@@ -1,28 +1,17 @@
...
 
 export interface AttachDataInterface {
-    sendLog(data: LogRQ): void;
-    sendAttachment(data: AttachmentRQ): void;
-    addDescription(data: string): void;
-    setAttribute(data: Attribute): void;
+    addAttributes(attributes: Array<Attribute>): void;
+    addDescription(text: string): void;
+    addLog(message: LogRQ['message'], level: LogRQ['level']): void;
+    addAttachment(message: LogRQ['message'], file: AttachmentRQ['file'], level: LogRQ['level']): void;
```
Comment (DiscoElevator) [inline src/realTimeReporter/reportingApi/attachData.ts]: `addAttachment(level: Pick<LogRQ, 'level'>, file: Pick<AttachmentRQ, 'file'>, message: Pick<LogRQ, 'message'>)`

File: src/realTimeReporter/reportingApi/attachData.ts [hunk 2]
```diff
@@ -1,28 +1,17 @@
...
-import { EVENTS } from '../../constants';
 
 export interface AttachDataInterface {
-    sendLog(data: LogRQ): void;
-    sendAttachment(data: AttachmentRQ): void;
-    addDescription(data: string): void;
-    setAttribute(data: Attribute): void;
+    addAttributes(attributes: Array<Attribute>): void;
+    addDescription(text: string): void;
+    addLog(message: LogRQ['message'], level: LogRQ['level']): void;
```
Comment (DiscoElevator) [inline src/realTimeReporter/reportingApi/attachData.ts]: `addLog(level: Pick<LogRQ, 'level'>, message: Pick<LogRQ, 'message'>): void;`

File: src/realTimeReporter/reportingApi/attachData.ts (hunk 1 above)
Comment (AmsterGet) [reply src/realTimeReporter/reportingApi/attachData.ts]: Fixed

File: src/realTimeReporter/reportingApi/attachData.ts (hunk 2 above)
Comment (AmsterGet) [reply src/realTimeReporter/reportingApi/attachData.ts]: Fixed

**Pull request merged**